plasmapy/particles/particle_collections.py

```diff
@@ -22,35 +20,34 @@
 
 class ParticleList(collections.UserList):
     """
-    A `list` like collection of
-    `~plasmapy.particles.particle_class.Particle` and/or
-    `~plasmapy.particles.particle_class.CustomParticle` objects.
+    A `list` like collection of particle objects.
```

Comment:
Why don't you want the links here?

Reply:
Good question. In retrospect, it does seem a better idea to link to `Particle` and `CustomParticle`, doesn't it?